RecoTracker/Configuration/python/customiseForQuadrupletsByCellularAutomaton.py

```diff
@@ -1,40 +1,90 @@
 import FWCore.ParameterSet.Config as cms
 
-def customiseForQuadrupletsByCellularAutomaton(process):
-    for module in process._Process__producers.values():
-        if not hasattr(module, "OrderedHitsFactoryPSet"):
-            continue
-	pset = getattr(module, "OrderedHitsFactoryPSet")
-        if not hasattr(pset, "ComponentName"):
-	    continue
-	if not (pset.ComponentName == "CombinedHitQuadrupletGenerator"):
-	    continue    
-        # Adjust seeding layers
-        seedingLayersName = module.OrderedHitsFactoryPSet.SeedingLayers.getModuleLabel()
+_CAParameters = dict(
+    maxChi2 = dict(
+        pt1    = 0.8, pt2    = 2,
+        value1 = 200, value2 = 100,
+        enabled = True,
+    ),
+    useBendingCorrection = True,
+    fitFastCircle = True,
+    fitFastCircleChi2Cut = True,
+    CAThetaCut = 0.00125,
+    CAPhiCut = 0.1,
+    CAHardPtCut = 0,
+)
+
+def customiseLegacySeeding(module):
+    pset = getattr(module, "OrderedHitsFactoryPSet")
```

Comment:
Full migration better be done sooner than later.
If we wait, we'll end up in the time scope of the policy that a default frozen menu has to be supported.

@fwyzard @Martin-Grunewald will we have to support the old PSet organization in 90X (starting from which point)?